src/main/java/com/mindee/geometry/Point.java

```diff
@@ -0,0 +1,38 @@
+package com.mindee.geometry;
+
+public final class Point {
+  private Double x;
+  private Double y;
+
+  public Point(double x, double y) {
+    this.x = x;
+    this.y = y;
+  }
+
+  /**
+   * @return the x
+   */
+  public Double getX() {
+    return x;
+  }
+
+  /**
+   * @return the y
+   */
+  public Double getY() {
+    return y;
+  }
+
+  @Override
```

Comment:
Lombok should be able to implement equals, toString, and hashcode methods for us. We do not need a custom implementation